crypter.go

```diff
@@ -74,8 +75,32 @@ type recipientKeyInfo struct {
 // EncrypterOptions represents options that can be set on new encrypters.
 type EncrypterOptions struct {
 	Compression CompressionAlgorithm
-	Type        ContentType
-	ContentType ContentType
+
+	// Optional map of additional keys to be inserted into the protected header
+	// of a JWS object. Some specifications which make use of JWS like to insert
+	// additional values here. All values must be JSON-serializable.
+	ExtraHeaders map[HeaderKey]interface{}
```

Comment:
This is an API change, but I think that's ok. I'll make a note to myself to bump the minor version when I tag this.